src/components/organisms/plans/plans.scss

```
@@ -0,0 +1,49 @@
+@use "../../variables" as *;
+.plans.container {
+  margin-top: 56px;
+  margin-bottom: 35px;
+  @media screen and (min-width: $breakpoint-lg) {
+    margin-bottom: 100px;
+  }
+
+  .message {
+    padding: 0px 20px 32px;
+    margin-bottom: 24px;
+    @media screen and (min-width: $breakpoint-lg) {
+      margin-bottom: 0px;
+    }
+    .title {
+      color: $orange_bistre;
+      margin-bottom: 0px;
+      padding-bottom: 6px;
+    }
+    .explain {
+      margin-bottom: 0px;
+    }
+  }
+  .wrapper {
+    margin-bottom: 24px;
+    @media screen and (min-width: $breakpoint-lg) {
+      display: flex;
+      justify-content: space-between;
+      margin-bottom: 40px;
+    }
+    .plan {
+      margin-bottom: 24px;
+      height: 100%;
+      @media screen and (min-width: $breakpoint-lg) {
+        width: 33%;
+        margin-bottom: 0px;
+      }
+      .list li:last-child {
+        @media screen and (max-width: $breakpoint-lg) {
```

```suggestion
        @media screen and (min-width: $breakpoint-lg) {
```
案件ではメディアクエリは統一して書くことが多いので、今回もそれに則ってこの形にしましょう。形を変更することで、書き方も修正が加わると思うので注意してください。